test/expressions/grouped-expressions.js

```diff
@@ -22,52 +22,13 @@ let testParseFailure = require('../assertions').testParseFailure;
 suite('Parser', () => {
   suite('grouped expressions', () => {
     // grouped expression that can be binding element and assignment target
-    testParse('(a)', expr, { type: 'IdentifierExpression', name: 'a' });
 
-    // grouped expression than cannot be binding element or assignment target
-    testParse('(0)', expr, { type: 'LiteralNumericExpression', value: 0 });
-
-    // mixture
-    testParse('(0, a)', expr, {
-      type: 'BinaryExpression',
-      left: { type: 'LiteralNumericExpression', value: 0 },
-      operator: ',',
-      right: { type: 'IdentifierExpression', name: 'a' },
-    });
 
-    testParse('(a, 0)', expr, {
-      type: 'BinaryExpression',
-      left: { type: 'IdentifierExpression', name: 'a' },
-      operator: ',',
-      right: { type: 'LiteralNumericExpression', value: 0 },
-    });
+    // grouped expression than cannot be binding element or assignment target
```

Comment:
Can you go through and remove/fix lonely comments like this one?

Reply:
Done.